scripting/pugsetup.sp

```diff
@@ -687,15 +687,25 @@ static void GiveReadyHints() {
 static void GiveCaptainHint(int client, int readyPlayers, int totalPlayers) {
   char cap1[MAX_NAME_LENGTH];
   char cap2[MAX_NAME_LENGTH];
-  if (IsPlayer(g_capt1))
+  if (IsPlayer(g_capt1)) {
     Format(cap1, sizeof(cap1), "%N", g_capt1);
-  else
+    if (strlen(cap1) > 14) {
```

Comment:
Can you extract the 14 to a constant?

Keeping it local to the function is fine, i.e.:
const int kMaxNameLength = 14;